package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua

```diff
@@ -9,6 +9,9 @@ function M.section(form)
   if enabled then
     local s = form:section(cbi.SimpleSection, nil,
       i18n.translate('This node will automatically update its firmware when a new version is available.'))
+  else
+    local s = form:section(cbi.SimpleSection, nil,
+      i18n.translate("Automatic updates are disabled. They can be enabled in <em>Advanced Settings</em>"))
```

Comment:
I don't remember, but I guess, i tested it. 
it already works for example in https://github.com/freifunk-gluon/gluon/blob/master/package/gluon-config-mode-contact-info/i18n/gluon-config-mode-contact-info.pot#L9